Gemfile.lock

```diff
@@ -31,7 +31,7 @@ GEM
     hashdiff (0.2.3)
     http-cookie (1.0.2)
       domain_name (~> 0.5)
-    json (1.8.1)
+    json (1.8.3)
```

Comment:
I was having issues with this version of json (ruby 2.3.0), the patch bump seemed to fix the installation.